src/jsonUtils/convert.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import type { SketchLayer } from '../types';
+
+// Converts a Sketch page object into it's JSON equivalent
+export const SketchToJSON = (sketchObj: SketchLayer): string => {
+  const imm = sketchObj.immutableModelObject();
+  return MSJSONDataArchiver.archiveStringWithRootObject_error_(imm, null);
+};
+
+// Converts a Sketch page object into it's JS Object equivalent
+export const SketchToJSObject = (sketchObj: SketchLayer): Object => {
+  const json = SketchToJSON(sketchObj);
+  return JSON.parse(json);
+};
+
+// Converts a JS Object tree into it's Sketch page object equivalent
+export const JSObjectToSketch = (jsTree: Object): SketchLayer => {
+  // prettier-ignore
+  const decodedData = MSJSONDictionaryUnarchiver
+    .unarchiveObjectFromDictionary_asVersion_corruptionDetected_error(
+      jsTree,
+      88, // 88 = Sketch v43+
```

Comment:
Should we pull this out as a constant to increment & name?

Reply:
Good call. Add `SKETCH_LOWEST_COMPATIBLE_VERSION` constant to track this going forward.